src/main/kotlin/app/hashers/CommitCrawler.kt

```diff
@@ -104,6 +111,18 @@ object CommitCrawler {
         df.setRepository(repo)
         df.setDetectRenames(true)
 
+        val confTreeWalk = TreeWalk(repo)
+        confTreeWalk.addTree(head.getTree())
+        confTreeWalk.setFilter(PathFilter.create(CONF_FILE_PATH))
+
+        var ignoredPaths =
+            if (confTreeWalk.next()) {
```

Comment:
I think this will read current repo HEAD, not default branch that we are selecting.